src/components/ManagerPage/AppsList.js

```diff
@@ -32,6 +38,7 @@ const ICONS_FALLBACK = {
 }
 
 type Status = 'loading' | 'idle' | 'busy' | 'success' | 'error'
+type Mode = '' | 'installing' | 'uninstalling'
```

Comment:
would be better to use `?Mode` and not have empty `''` as a possible type . or something more meaningful like 'home'